### PR TITLE
New: Unicorn.PaintAid version 2.1.0.0

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25726)

### CODE REVIEW

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Unicorn.PaintAid
+PackageVersion: 2.1.0.0
```
Comment (ItzLevvie) [inline manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.9.1
```

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Unicorn.PaintAid
+PackageVersion: 2.1.0.0
```
Comment (ItzLevvie) [inline manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.yaml:5]:
```suggestion
PackageVersion: 2.0.9.1
```

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Unicorn.PaintAid
+PackageVersion: 2.1.0.0
```
Comment (ItzLevvie) [inline manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 2.0.9.1
```

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.yaml:5]: Windows Package Manager requires this key to match Control Panel.

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.zh-CN.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.zh-CN.yaml:5]: Windows Package Manager requires this key to match Control Panel.

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.installer.yaml:5]: Windows Package Manager requires this key to match Control Panel.

File: manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: Unicorn.PaintAid
+PackageVersion: 2.1.0.0
```
Comment (ItzLevvie) [inline manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.9.1
```

Comment (ItzLevvie) [reply manifests/u/Unicorn/PaintAid/2.1.0.0/Unicorn.PaintAid.locale.en-US.yaml:5]: Windows Package Manager requires this key to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Oops, wait. I might have checkout the wrong PR. Let me double check.

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
I forgot to mention that Unicorn.UDMPaint would install an older version of Unicorn.PaintAid.🙊

**Review by ghost — APPROVED** [review]

**Pull request merged**